main.js

```diff
@@ -17,7 +17,10 @@ const {
   EVENT_RUN_END,
   EVENT_TEST_FAIL,
   EVENT_TEST_PASS,
-  EVENT_SUITE_BEGIN,
+  EVENT_TEST_PENDING,
+  EVENT_TEST_BEGIN,
+  EVENT_TEST_RETRY,
```

Comment:
Are these being used?

Reply:
not anymore, the linter should have caught this, but didn't for some reason so thanks for catching this: https://github.com/digitalbazaar/mocha-w3c-interop-reporter/pull/7/commits/4585e53dd271bcf777588d558747dd6eb3366102